draft-west-cookie-incrementalism.md

```diff
@@ -224,6 +233,59 @@ attribute (Section 4.1.2.5 of {{RFC6265bis}}) by altering the storage model defi
 This is conceptually similar to the requirements put into place for the `__Secure-` prefix (Section
 4.1.3.1 of {{RFC6265bis}}).
 
+## Schemeful Same-Site {#schemeful-samesite}
+
+By using the scheme, as well as the registrable domain, SameSite can help to
+protect https origins against a network attacker that is impersonating an http
+origin with the same registrable domain. Further increasing its CSRF
+protections. To do so we need to modify a number of things:
```

Comment:
Looks like you didn't finish this sentence? (Further [...])

Reply:
Just a too casual take on the wording. I've switched things around a bit, LMK.

Reply:
@davidben I'm going to merge this PR and open a PR on mikewest's repo in the name of progress. If you still have any comments please open them there.